### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by sk-** [actgardner/gogen-avro#206]
Hi @actgardner, I'm creating this issue to confirm if this project is still being maintained. It has served us well enough, but there are many open issues that are affecting us (like UUID support, the representation of nullable fields, name of fields, etc), which are making us consider this dependency.

The fact that there have been no commits, no responses to either Issues or Pull requests, indicates me this probably abandoned.

It's completely valid to acknowledge this is no longer maintained, but it'd good be good  to make it explicit for everyone using or planning to use it.

**Comment by actgardner** [comment]
Hi @sk-. I've been maintaining this project on-and-off for almost 5 years. I wouldn't consider it "abandoned" because I do sometimes come back to add new feature requests or implement a new optimization if I have the time and energy. That said:

- I don't have a timeline for logical types. I would be very open to a PR adding them that had tests.
- Nullable unions are a constant struggle with Go's type system (pre-generics).
- Implementing a general fix for name collisions (support for custom name patterns?) would be great, and I would accept a PR adding them. I'm not open to specific fixes that only work for some subset of names.

If I happen to be on a flight and get inspired I might implement one of these. You're also welcome to fork the project and take it in a direction you prefer (if you remove some of the features the null unions get easier, for instance).

I've been away long enough that Go generics have also landed. Completely reworking the library to support generics is unlikely to ever happen.

edit: I've added a disclaimer to the README.

edit 2: If you do decide to fork the project I'm happy to redirect people to a better-maintained version.

**Comment by tanis2000** [comment]
@actgardner thanks a lot for all the effort you put into this project. The company I am working with is heavily relying on this library for the backbone of a huge project that spans over multiple years. I am pretty sure we do not want to migrate to a different implementation.

I would suggest to add more people that you trust to the list of the maintainers so that PRs will still be managed and you will also get a bit of lift from the heavy load of maintaining this kind of project.

I just noticed that 2 weeks ago a PR that we also need has landed into the main branch but there is no tag for it. It would be nice to get a new patch release in the near future.

**Comment by tomasfabian** [comment]
I agree, it would be great if at least one of the contributors could join you as a maintainer.
